Foundation/Process.swift

```diff
@@ -318,12 +374,11 @@ open class Process: NSObject {
         }
 
 #if os(Windows)
-        // TODO(compnerd) quote the commandline correctly
-        // https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/
         var command: [String] = [launchPath]
         if let arguments = self.arguments {
           command.append(contentsOf: arguments)
         }
+        command = command.map(quoteWindowsCommandArg)
```

Comment:
Can you please keep the transformation of the `[String]` input to the final `String` passed to the native API, localized to a single function call? Like:

```swift
private func quoteWindowsCommandLine(_ commandLine: [String]) -> String {
    func quoteWindowsCommandArg(arg: String) -> String { /* ... */ }
    return commandLine.map(quoteWindowsCommandArg).joined(separator: " ")
}
```